.github/workflows/build-node-examples.yml

```diff
@@ -0,0 +1,110 @@
+name: build node examples from source
+
+on: pull_request
+
+jobs:
+  node:
+    runs-on: ${{ matrix.platform }}
+    timeout-minutes: 40
+
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu-latest, macos-latest, windows-latest]
+        example:
+          - name: GatsbyThemedSite
+            folder: react/gatsby-themed-site
+            executable: gatsby-themed-site-app
+          - name: CRAApp
+            folder: react/create-react-app
+            executable: app
+          - name: NextjsApp
+            folder: react/next.js
+            executable: app
+          - name: VanillajsMonolithApp
+            folder: vanillajs/monolith
+            executable: app
+          - name: quasar-app
+            folder: vue/quasar-app
+            executable: app
+          - name: svelte-app
+            folder: svelte/svelte-app
+            executable: app
+        include:
+          - platform: ubuntu-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release/bundle/deb
+            ext: _0.1.0_amd64.deb
+          - platform: macos-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release/bundle/osx
+            ext: .app
+          - platform: windows-latest
+            CARGO_HOME: ~/.cargo
+            releaseFolder: target/release
+            ext: .x86.msi
+
+    steps:
+      - name: checkout tauri
+        uses: actions/checkout@v2
+        with:
+          repository: tauri-apps/tauri
+          ref: dev
+      - name: checkout examples
+        uses: actions/checkout@v2
+        with:
+          path: examples
+      - name: install rust stable
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+      - name: install webkit2gtk (ubuntu only)
+        if: matrix.platform == 'ubuntu-latest'
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y webkit2gtk-4.0
+      - name: cache rust bin
+        uses: actions/cache@v1
+        with:
+          path: ${{ format('{0}/bin/', matrix.CARGO_HOME) }}
+          key: ${{ runner.OS }}-build-bin-${{ hashFiles('**/Cargo.toml') }}-
+      - name: cache rust registry/index
+        uses: actions/cache@v1
+        with:
+          path: ${{ format('{0}/registry/index/', matrix.CARGO_HOME) }}
+          key: ${{ runner.OS }}-build-reg-index-${{ hashFiles('**/Cargo.toml') }}-
+      - name: cache rust registry/cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ format('{0}/registry/cache/', matrix.CARGO_HOME) }}
+          key: ${{ runner.OS }}-build-reg-cache-${{ hashFiles('**/Cargo.toml') }}-
+      - name: build rust
+        run: cargo build
+        env:
+          TAURI_DIST_DIR: ../../test/fixture/dist
```

Comment:
this looks wrong

Reply:
Hmm, are you thinking this needs another folder level as well?

Reply:
Changing that doesn't seem to make a difference.
https://github.com/tauri-apps/examples/pull/7/checks?check_run_id=443575019#step:15:10